multi-buffer-search/src/commands/replaceCommand.ts

```
@@ -0,0 +1,129 @@
+            editor,
+            mapping,
+            options
+          )
+          const fileCount = results.size
+          const matchCount = Array.from(results.values()).reduce(
+            (sum, excerpts) => sum + excerpts.length,
+            0
+          )
+          vscode.window.showInformationMessage(
+            `Ready to replace ${matchCount} matches in ${fileCount} files. ` +
+              `Review changes and press Ctrl+S to apply.`
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)